Tests/AppTests/AppTests.swift

```diff
@@ -1,13 +1,258 @@
+                                },
+                                {
+                                  "type" : "text",
+                                  "text" : " Commit 3"
+                                }
+                              ]
+                            }
+                          ]
+                        },
+                        {
+                          "type" : "listItem",
+                          "content" : [
+                            {
+                              "type" : "paragraph",
+                              "content" : [
+                                {
+                                  "type" : "inlineCard",
+                                  "attrs" : {
+                                    "url" : "https:\/\/babylonpartners.atlassian.net:443\/browse\/CRP-4142#icft=CRP-4142"
+                                  }
+                                },
+                                {
+                                  "type" : "text",
+                                  "text" : " Commit 4"
+                                }
+                              ]
+                            }
+                          ]
+                        }
+                      ]
+                    }
+                  ],
+                  "version" : 1
+                },
+                "customfield_11512" : {
+                  "type" : "doc",
+                  "content" : [
+                    {
+                      "type" : "paragraph",
+                      "content" : [
+                        {
+                          "type" : "text",
+                          "text" : "TBD"
+                        }
+                      ]
+                    }
+                  ],
+                  "version" : 1
+                },
+                "project" : {
+                  "id" : "13402"
+                },
+                "customfield_12541" : "https:\/\/github.com\/company\/project\/releases\/tag\/app\/1.2.3",
+                "customfield_11505" : {
+                  "name" : "andreea.papillon"
+                }
+              }
+            }
+            """#
+
+        if (issueJson != expectedTicketJson) {
+            let lhsLines = (issueJson?.split(separator: "\n") ?? []).enumerated()
+            let rhsLines = expectedTicketJson.split(separator: "\n").enumerated()
+            let diff = zip(lhsLines, rhsLines)
+                .filter { (lhs, rhs) in lhs.element != rhs.element }
+                .map { (lhs, rhs) in "\(lhs.offset): |\(lhs.element)| <> |\(rhs.element)|"}
+            XCTFail("Ticket JSON Mismatch – see diff:\n\(diff.joined(separator: "\n"))")
+        }
+    }
+
+    #if !os(Linux)
+    private func attachment(name: String, string: String?) -> XCTAttachment {
+        let attachment = XCTAttachment(string: string ?? "<nil>")
+        attachment.name = name
+        return attachment
     }
+    #endif
 
     static let allTests = [
```

Comment:
I think `swift test --generate-linuxmain` put this in a separate file now so we can remove it from here